webchat/auth/loginCard.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function() {
+  function LoginCardController(AuthService, MessageService) {
+    const ctrl = this;
+    ctrl.user = {};
+    ctrl.isLoadingUser = () => AuthService.isLoadingUser;
+
+    ctrl.signIn = () => {
+      return AuthService.loginWithEmailAndPassword(ctrl.user.email, ctrl.user.password).then(() => {
+        ctrl.onLogin();
+      }).catch((e) => {
+        MessageService.showToast(e);
+      });
+    }
+
+    ctrl.loginWithGoogle = () => {
+      return AuthService.loginWithGoogle().then(() => {
+        ctrl.onLogin();
+      }).catch((e) => {
+        MessageService.showToast(e);
+      });
+    }
+
+    ctrl.resetPassword = () => {
+      window.open(Config.FRONTEND_URL);
```

Comment:
Is the correct URL?

Reply:
Yes. We won't handle password reset on webchat, but redirect to frontend so the user can use the reset feature there.